common_utils/shareMem.go

```diff
@@ -0,0 +1,58 @@
+package common_utils
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	memKey = 7749
+	memSize = 1024
+	memMode = 01600
```

Comment:
>memMode [](http://example.com/codeflow?start=1&length=7)

Add comment

Reply:
updated

Reply:
You did not mention "memMode" in above comment.

Reply:
I have updated the comment for memMode.